packages/wix-storybook-utils/src/Sections/views/table.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { TableSection } from '../../typings/story-section';
+
+const Markdown = require('../../Markdown').default;
+
+const styles = require('./styles.scss');
+
+export const table: (a: TableSection) => React.ReactNode = ({ rows }) => {
+  return (
+    <table className={styles.table}>
+      <tbody>
+        {rows.map((row, i) => (
+          <tr key={`tr-${i}`} className={styles.tr}>
+            {row.map((cell, j) => (
+              <td key={`td-${i}-${j}`} className={styles.td}>
+                <Markdown source={cell} className="table-markdown" />
```

Comment:
let's allow react nodes in here too
```suggestion
                { typeof cell === 'string' ? <Markdown source={cell} className="table-markdown" /> : cell }
```

this way giving things like `<LinkTo/>` inside cell would be possible